definitions/variable/technology/technologies.yaml

```
@@ -4,6 +4,9 @@
 - Capacity|Electricity|{Electricity Input}:
     description: Total installed electricity generation capacity from {Electricity Input}
     unit: GW
+- Capacity|Electricity|{Energy Storage System Type}:
```

```suggestion
- Capacity|Electricity|Energy Storage System:
    description: Total installed electricity generation capacity from all {Energy Storage System Type} storage systems
    unit: GW
- Capacity|Electricity|Energy Storage System|{Energy Storage System Type}:
```
For consistency with the other variables related to energy storage, and also use of the aggregations
I also propose to include the aggregated capacity of all batteries

---

Sounds good to me - potentially, I would name it 
Capacity|Electricity|Energy Storages
since that sounds nicer in my opinion. But if this is the naming used everywhere else, then we can of course use this.

Also, yes, it would then also provide an aggregated number of all electricity storages.

---

@loeffko I would have agreed for Energy Storages but the other name is used everywhere so.....

---

@sandrinecharousset done. Should be fine now.